### DISCUSSION TITLE
С любого из сайтов не скачивается

### DISCUSSION THREAD

**Discussion by markuslement** [lirix360/ReadmangaGrabber#68] [Q&A]
https://readmanga.live/novaia_jizn_ubiicy_bogov__A533b
Вот 1 из примеров 
Создаются папки , если выбрать создание PDF или CBZ то и файлы создаются 
Проблема : Файлы пустые , вес 1кб
Папки тоже пустые 
Как решить ?

**Comment by lirix360** [comment]
[Версия 2.1.2](https://github.com/lirix360/ReadmangaGrabber/releases/tag/v2.1.2), доступная в релизах, все прекрасно скачивает.

**Reply by markuslement** [reply]
> [Версия 2.1.2](https://github.com/lirix360/ReadmangaGrabber/releases/tag/v2.1.2), доступная в релизах, все прекрасно скачивает.

![image](https://user-images.githubusercontent.com/50837114/205641381-f287722c-03f4-4bd5-9e2e-79366b68ccda.png)


Новая версия , скачивает якобы за доли секунды, и просто создает файлы и папки(

**Reply by lirix360** [reply]
Исправления регулярки есть только в скомпилированных версиях, в исходниках их нет на данный момент.

**Reply by markuslement** [reply]
> Исправления регулярки есть только в скомпилированных версиях, в исходниках их нет на данный момент.

Спасибо!
Действительно, в готовой версии все работает )

**Reply by PredatoryWerewolf** [reply]
> [Версия 2.1.2](https://github.com/lirix360/ReadmangaGrabber/releases/tag/v2.1.2), доступная в релизах, все прекрасно скачивает.

https://mintmanga.live/smertelnaia_igra_mertvoi_gory
Данную мангу не скачивает. Новая версия также создает пустые папки и PDF, CBZ пусты.

**Reply by lirix360** [reply]
> https://mintmanga.live/smertelnaia_igra_mertvoi_gory Данную мангу не скачивает. Новая версия также создает пустые папки и PDF, CBZ пусты.

Для просмотра/скачивания данной манги обязательна авторизация на сайте, скачивание такой манги на данный момент не поддерживается.
